apps/client/server/profile/mutations.ts

```diff
@@ -0,0 +1,13 @@
+import type { UseQueryOptions } from "@tanstack/react-query";
+import { useMutation } from "@tanstack/react-query";
```

Comment:
```suggestion
import { useMutation, type UseQueryOptions } from "@tanstack/react-query";
```

한줄로도 가능합니다! 개바개이므로 입맛에 맞으시면 사용해주세요!

Reply:
저도 한줄이 좋은데 vscode가 말을 안듣네요 ㅠㅠ ㅋ감사합니다

Reply:
[fix: type import one line](https://github.com/toppingskorea/toppings-client/pull/40/commits/d299ca5966801e3da44cd23e2ca25e7f03bded64)